lib/biokbase/workspace/baseclient.py

```diff
@@ -125,7 +121,7 @@ def __init__(
             self, url=None, timeout=30 * 60, user_id=None,
             password=None, token=None, ignore_authrc=False,
             trust_all_ssl_certificates=False,
-            auth_svc='https://kbase.us/services/auth/api/legacy/KBase/Sessions/Login',
+            auth_svc='https://kbase.us/services/authorization/Sessions/Login',
```

Comment:
It looks like this was compiled with a very old version of kb-sdk, this is a reversion

Reply:
Ditto.

Reply:
:+1: